### ISSUE TITLE
`useDroppable` doesn't work on root component

### ISSUE THREAD

**Issue by tsuharesu** [clauderic/dnd-kit#1356]
https://codesandbox.io/p/sandbox/magical-mountain-t2fmm7?file=%2Fsrc%2FApp.tsx

It took me a *long long* time to understand why my droppable was not working, and it seems that the hook doesn't work correctly when it is in the same level as `DndContext`.

**Comment by jitender-techindustan** [comment]
yes i was facing the same issue, having the droppable at the same level where `DndContext` is not working.

**Comment by a1782680475** [comment]
我也是这个问题，花了一下午时间才解决！

**Comment by clauderic** [comment]
This issue was opened against a previous version of dnd-kit. The library has since undergone a major rewrite.

It has been rebuilt from the ground up with support for **multiple frameworks** (React, Vue, Svelte, Solid, and vanilla JavaScript), improved performance, and a new API.

- **Documentation**: https://dndkit.com
- **Migration guide**: https://dndkit.com/react/migration
- **GitHub**: https://github.com/clauderic/dnd-kit

If this issue is still relevant to the latest version, please open a new issue.
